File: task4.py
def task4(number):
    print("  ", end=" ")
    for i in range(1, number + 1):
        print(f"{i:2}", end=" ")
    print()
    for i in range(1, number + 1):
        print(f"{i:2}", end=" ")
        for j in range(1, number + 1):
            print(f"{(i * j):2}", end=" ")
        print()

File: task4.js
function task4(number) {
    process.stdout.write("  ");
    for (let i = 1; i <= number; i++) {
        process.stdout.write(` ${i.toString().padStart(2, ' ')} `);
    }
    console.log();
    for (let i = 1; i <= number; i++) {
        process.stdout.write(`${i.toString().padStart(2, ' ')} `);
        for (let j = 1; j <= number; j++) {
            process.stdout.write(`${(i * j).toString().padStart(2, ' ')} `);
        }
        console.log();
    }
}